pkg/tests/install/test.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python3
+
+# Copyright 2021 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import os
+import unittest
+import stat
+import subprocess
+
+from rules_python.python.runfiles import runfiles
+from private.manifest import ENTRY_IS_FILE, ENTRY_IS_LINK, ENTRY_IS_DIR, ManifestEntry, entry_type_to_string
```

Comment:
Why don't you just import manifest and qualify the uses later. That will make it easier as we add more things (like ENTRY_IS_EMPTY_FILE, which I have in another PR).

Reply:
Done in e95bdc8.